app/src/app/[lng]/cities/[cityId]/HIAP/page.tsx

```diff
@@ -61,72 +76,13 @@ export default function HIAPPage(props: {
             flexDirection="column"
             gap="24px"
           >
-            {/* citycatalyst actions section */}
-            <Box display="flex" flexDirection="column" gap="24px" pb="24px">
-              <Box display="flex" flexDirection="column" gap="16px">
-                <Text
-                  color="content.link"
-                  fontFamily="heading"
-                  fontSize="title.sm"
-                  fontWeight="bold"
-                  textTransform="uppercase"
-                >
-                  {t("citycatalyst-actions-title")}
-                </Text>
-              </Box>
-              <Box
-                display="flex"
-                flexDirection="row"
-                gap="24px"
-                alignItems="center"
-              >
-                <Box display="flex" flexDirection="column" gap="8px">
-                  <Text
-                    fontFamily="heading"
-                    fontWeight="bold"
-                    fontSize="headline.sm"
-                    color="content.secondary"
-                  >
-                    {t("top-actions-for-your-city")}
-                  </Text>
-                  <Text
-                    fontFamily="body"
-                    fontSize="body.lg"
-                    color="content.tertiary"
-                  >
-                    {t("top-actions-for-your-city-description")}
-                  </Text>
-                </Box>
-                <Box>
-                  <Button
-                    bg="content.link"
-                    color="white"
-                    px="24px"
-                    h="84px"
-                    borderRadius="16px"
-                    gap="12px"
-                    onClick={() => {
-                      // TODO: add logic to re-prioritize actions
-                    }}
-                  >
-                    <Icon
-                      as={LuRefreshCw}
-                      rotate={"270deg"}
-                      boxSize={"36px"}
-                      color="white"
-                    />
-                    <Text
-                      fontFamily="heading"
-                      fontWeight="bold"
-                      fontSize="headline.sm"
-                      textTransform="none"
-                    >
-                      {t("re-prioritize-actions")}
-                    </Text>
-                  </Button>
-                </Box>
-              </Box>
-            </Box>
+            <ClimateActionsSection
+              t={t}
+              actions={hiapData}
+              onReprioritize={() => {
+                // TODO: add logic to re-prioritize actions
```

Comment:
Can you add a ticket for the next sprint for this? Sorry for the delay regarding the API.